internal/rest/avi_evh_obj.go

```diff
@@ -492,8 +491,9 @@ func (rest *RestOperations) AviVsChildEvhBuild(vs_meta *nodes.AviEvhVsNode, rest
 		Enabled:               vs_meta.Enabled,
 		VhType:                proto.String(utils.VS_TYPE_VH_ENHANCED),
 	}
-	if lib.GetT1LRPath() == "" {
-		evhChild.VrfContextRef = &vrfContextRef
+
+	if vs_meta.VrfContext != "" {
```

Comment:
we need to check AKO run with NSXT deployment to avoid regression. Assuming here we have taken care setting of VrfContext properly at graph layer.